addons/website/static/src/builder/plugins/form/utils.js

```diff
@@ -156,7 +156,11 @@ export function renderField(field, resetId = false) {
     template.content.querySelectorAll("[data-name]").forEach((el) => {
         el.dataset.name = getQuotesEncodedName(el.dataset.name);
     });
-    return template.content.firstElementChild;
+    const fieldEl = template.content.firstElementChild;
+    if (field.formCopyEmail) {
+        fieldEl.classList.add("s_website_form_copy_email");
+    }
```

Comment:
IF this logic will only be executed when you're calling `renderField` from [`setSendCopyEmailField`](https://github.com/odoo/odoo/pull/219914/files#diff-42acf7472f772590acbe49f35909bb0060f78e5b7db5414ce504a6df38c795dbR484), you could consider shifting this block there as it's specific to your code and this function will remain generic.

Reply:
Yes. When we drop a snippet that contains a website form, the email field loses the `s_website_form_copy_email` class due to the field being re-rendered. Since we want this feature enabled by default, the code also handles this scenario in addition to the cloning case.

Reply:
consider simplifying this
```suggestion
    template.content.firstElementChild.classList.toggle("s_website_form_copy_email", !!field.formCopyEmail);
```